LIB_DOC.md

```diff
@@ -0,0 +1,246 @@
+Subxt is a library to **sub**mit e**xt**rinsics to a [substrate](https://github.com/paritytech/substrate) node via RPC.
+
+The generated Subxt API exposes the ability to:
+- [Submit extrinsics](https://docs.substrate.io/v3/concepts/extrinsics/) (Calls)
+- [Query storage](https://docs.substrate.io/v3/runtime/storage/) (Storage)
+- [Query constants](https://docs.substrate.io/how-to-guides/v3/basics/configurable-constants/) (Constants)
+- [Subscribe to events](https://docs.substrate.io/v3/runtime/events-and-errors/) (Events)
+
```

Comment:
I like these examples, but I'm not sure about having them in the readme here, because they aren't statically checked and such. I wonder whether we should just be pointing to the examples folder more prominently, since those are more likely to remain uptodate; what do you think?